site/public/css/course-materials.css

```
@@ -14,3 +14,8 @@
     color: var(--standard-dark-green);
     float: none;
 }
+
+a.popout-item:hover *,
+a.popout-item:hover {
```

Looking at this, I think we want the selectors to be:
```suggestion
a.popout-item:hover,
a.popout-item:hover *,
a.popout-item > .fas:hover {
```

You're right we need the `a.popout-item:hover`, which I missed in my original suggestion and just happened to overlapped the color in `.file-viewer a:hover` so I didn't see that we needed this.

---

yup it looks good now